advisories/a88042.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Technical Advisory 88042
+advisory: A-88042
+summary: <code>RESTORE</code> of incremental backups, which were created during a specific series of events relating to <code>IMPORT INTO</code> and cancellation, may include rows that should not be restored.
+toc: true
+affected_versions: v22.1.0 to v22.1.8
+
+advisory_date: 2022-10-24
+docs_area: releases
+---
+
+Publication date: {{ page.advisory_date | date: "%B %e, %Y" }}
+
+## Description
+
+In CockroachDB [v22.1.0](../releases/v22.1.html#v22-1-0) to [v22.1.8](../releases/v22.1.html#v22-1-8), `RESTORE` of incremental backups, which were created during a specific series of events relating to `IMPORT INTO` and cancellation, may include rows that should not be restored. This has been fixed in CockroachDB patch version v22.1.9, available in [Releases](https://www.cockroachlabs.com/docs/releases/).
```

Comment:
nit: recommend an article for all mentions of `RESTORE` overloaded as a conceptual task as well as its literal reference statement. Here:

> a `RESTORE` of ...

nit: `which` could loosely apply to either `RESTORE` or `incremental backups`. Here's a more literal approach, using `certain` to frontload concept "only applies to a subset".

> a `RESTORE` of certain incremental backups that had been created during a specific series of events relating to `IMPORT INTO` and cancellation may include rows that ...

Or, with parens and `those` for even more clarity (since trigger case includes more than one condition, and begins to bleed into surrounding sentence):

> a `RESTORE` of certain incremental backups (those that had been created during a specific series of events relating to `IMPORT INTO` and cancellation) may include rows that ...

Reply:
nit: This would be a good place to link the following, if you were so inclined:
```
[`RESTORE`](../{{site.versions["stable"]}}/restore.html)
[`IMPORT INTO`](../{{site.versions["stable"]}}/import-into.html)
```